main.go

```diff
@@ -222,24 +222,28 @@ func main() {
 		log.Fatalf("Unable to initialize the rasterizer cache: %s", err)
 	}
 
-	// Set up an S3-backed filecache to underly the rasterCache
-	fCache, err := filecache.NewS3Cache(
-		config.CacheSize, config.BaseDir, config.AwsRegion, ServerWriteTimeout,
+	// Set up a filecache to underly the rasterCache
```

Comment:
This is now able to handle both S3 and Dropbox downloads simultaneously. It will be able to still fetch Dropbox files if the S3 credentials are not set.